inject-generator/src/main/java/io/avaje/inject/generator/BeanReader.java

```diff
@@ -264,14 +264,12 @@ private void readMethod(Element element, boolean factory) {
       }
     }
 
-    if (context.isPostConstructAvailable()) {
```

Comment:
Thanks for spotting that ... ah, tests passed because it was still in the classpath - nice !!